src/scripts/main.js

```diff
@@ -2,7 +2,20 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
 
-// write your code here
+const newPeople = [...people];
+
+newPeople.forEach(person => {
+  const cells = document.createElement('tr');
+
+  cells.insertAdjacentHTML('afterbegin', `
+  <td>${person.name}</td>
+  <td>${person.sex === 'm' ? 'Male' : 'Female'}</td>
+  <td>${person.born}</td>
+  <td>${person.died}</td>
+  <td>${person.age = person.died - person.born}</td>
```

Comment:
no need to define the new field in `person` object in this case
also, it looks a bit strange inside the string template HTML)

Reply:
I agree. Here you create a new property of an object inside the HTML template. if you want you can do it before the template or just use `person.died - person.born`.  And check code style (indents).

Reply:
I'm not sure, that I finally understand you correctly, but I hope so)))